src/wallet/wallet2.h

```diff
@@ -789,16 +791,16 @@ namespace tools
       src.amount = td.amount();
       src.rct = false;
       //paste mixin transaction
-      if(daemon_resp.outs.size())
+      if(amounts_with_outputs.size())
       {
-        daemon_resp.outs[i].outs.sort([](const out_entry& a, const out_entry& b){return a.global_amount_index < b.global_amount_index;});
-        BOOST_FOREACH(out_entry& daemon_oe, daemon_resp.outs[i].outs)
+        std::sort(amounts_with_outputs[i].outputs.begin(), amounts_with_outputs[i].outputs.end(), [](const cryptonote::rpc::output_key_and_amount_index& a, const cryptonote::rpc::output_key_and_amount_index& b){return a.amount_index < b.amount_index;});
+        BOOST_FOREACH(cryptonote::rpc::output_key_and_amount_index& daemon_oe, amounts_with_outputs[i].outputs)
         {
-          if(td.m_global_output_index == daemon_oe.global_amount_index)
+          if(td.m_global_output_index == daemon_oe.amount_index)
```

Comment:
Is there a raeson for gratuitous renaming ? While the new name isn't objectoinable, it all adds up to remembering that something else is what used to be someting else.